src/main/resources/com/puppycrawl/tools/checkstyle/checks/imports/messages.properties

```diff
@@ -1,4 +1,5 @@
 custom.import.order=Import statement for ''{2}'' is in the wrong order. Should be in the ''{0}'' group, expecting group ''{1}'' on this line.
+custom.import.order.groups.separated.internally=Extra separation in import group before ''{0}''
```

Comment:
Please make this a complete sentence. Other languages will have to be updated as well.

Reply:
As the case, check line separator with import group, is the same with `ImportOrderCheck`,  the `custom.import.order.groups.separated.internally` was a duplicate of `import.groups.separated.internally` . Other languages were duplicated as well.

> Please make this a complete sentence.

I'm not sure how to describe the error message precisely, suggestions are welcome :)

Reply:
Lets leave it as it is then for now.